src/main/webapp/dataset.xhtml

```diff
@@ -1867,16 +1868,19 @@
                     
                     function testTOADatasetPage() {
                         var termsofAccessHidden, fileAccessRequestHidden;
+                        var hasRestricted;
                         try{
                             termsofAccessHidden = document.getElementById("datasetForm:tabView:termsofAccessHiddenLT").value;
                             fileAccessRequestHidden  = document.getElementById("datasetForm:tabView:fileAccessRequestHiddenLT").value;  
                         }
                         catch (error){
+                            alert("notdatasetform");
```

Comment:
this seems like it should not be here :)

Reply:
good catch, thanks